spec/fixtures/education_benefits_claims/1990/kitchen_sink.spl

```diff
@@ -114,7 +114,7 @@ Active Duty    From Active Duty
 2013-04-22     2013-06-14         Army Reserve
                          Service Status:   ACTIVE DUTY
                          Involuntary Call: yes
-Benefits to Apply to: chapter33, chapter1606,
+                         Benefits to Apply to: chapter33, chapter1606,
```

Comment:
this is what the template should be generating, but before Rails 6, this wan't including these leading whitespaces. https://github.com/department-of-veterans-affairs/vets-api/blob/15c3a5cd53a6b1b7c4a28e8c5653cf3c67737f47/app/workers/education_form/templates/tours_of_duty.erb